analysis_options.yaml

```
@@ -1,4 +1,4 @@
-include: package:pedantic/analysis_options.yaml
+include: package:lints/recommended.yaml
```

Based on below package suggestion, we should change that to:

```suggestion
include: package:flutter_lints/flutter.yaml
```